kythe/web/site/examples.md

```diff
@@ -30,13 +30,13 @@ mkdir -p "$KYTHE_OUTPUT_DIRECTORY"
 #   com.google.devtools.kythe.extractors.java.standalone.Javac8Wrapper \
 #   <javac_arguments>
 java -Xbootclasspath/p:third_party/javac/javac*.jar \
-  com.google.devtools.kythe.extractors.java.standalone.Javac8Wrapper \
-  kythe/java/com/google/devtools/kythe/common/*.java
+  -jar /opt/kythe/extractors/javac_extractor.jar \
+  com.google.devtools.kythe.extractors.java.standalone.Javac9Wrapper \
```

Comment:
Javac8Wrapper

Reply:
That jar is the JDK 9 API built to run on JRE 8.  Likewise, Javac8Wrapper is built to run on JRE 8.  The other jars in third_party/javac and Javac9Wrapper are experimentally built for JRE 9.

Reply:
Ahh makes sense. I changed it back to Javac8Wrapper.